### DISCUSSION TITLE
Before and After update action

### DISCUSSION THREAD

**Discussion by theincogtion** [Devolutions/UniGetUI#2935] [Q&A]
unigetui fails to upgrade syncthingtray because its running in background.
Is it possible to stop an application before the update and execute it afterwards?

**Comment by marticliment** [comment]
This issue should be handled either by the installer of syncthingtray or, in its defect, by the package manager from which you are trying to update the program

**Reply by theincogtion** [reply]
I am using winget. I raised an issue here:
https://github.com/Martchus/syncthingtray/issues/291
Nevertheless if there would be an option for pre and post tasks I could stop and start it.

**Reply by marticliment** [reply]
Pre/Post-install commands are something that has been requested, but I am not 100% sure if I am going to implement them, since they could pose a security risk to the user (they could be used for rights escalation under certain circumstances)
